### PR TITLE
Feature/wysiwyg comments

### PR DESCRIPTION
**Pull request by ScuffedNewt**

### CODE REVIEW

File: app/Http/Controllers/Comments/CommentController.php [hunk 1]
```diff
@@ -89,7 +89,7 @@ public function store(Request $request, $model, $id) {
         }
 
         $comment->commentable()->associate($base);
-        $comment->comment = $request->message;
+        $comment->comment = config('lorekeeper.settings.wysiwyg_comments') ? $request->message : parse($request->message);
```
Comment (itinerare) [inline app/Http/Controllers/Comments/CommentController.php:92]: This needs to be run through HTML purifier if WYSIWYG I think (with any needed changes to edit logging as well).

**Review by itinerare — CHANGES_REQUESTED** [review]
Looks good, though I'd like to see it bolted down a little more to be safe.

File: app/Http/Controllers/Comments/CommentController.php (hunk 1 above)
Comment (Ne-wt) [reply app/Http/Controllers/Comments/CommentController.php:92]: parse is the purifier

Comment (Ne-wt) [reply app/Http/Controllers/Comments/CommentController.php:92]: also edit logging has been updated already

**Review by itinerare — APPROVED** [review]

**Review by AW0005 — APPROVED** [review]

**Pull request merged**